packages/eslint-config/index.js

```diff
@@ -9,10 +7,12 @@ const config = {
     'plugin:import/errors',
     'plugin:import/warnings',
     'plugin:you-dont-need-lodash-underscore/compatible',
-    'plugin:prettier/recommended', // This has to be the last rule added.
+    'prettier',
   ],
   plugins: ['node', 'unicorn'],
   rules: {
+    'arrow-body-style': 'off', // This rule is handled by our Prettier config.
```

Comment:
for posterity, we were previously disabling this rule with `eslint-plugin-prettier` but since this PR removes that, we need to do it manually now: 

https://github.com/prettier/eslint-config-prettier/blob/5687e7de69ac132f053b4bc5474d5963aa3bf6d6/README.md#arrow-body-style-and-prefer-arrow-callback
https://github.com/prettier/eslint-config-prettier/issues/31
https://github.com/prettier/eslint-config-prettier/issues/71

🫠

Reply:
```suggestion
    'arrow-body-style': 'off', // This rule clashes with our Prettier config.
```